dbms/src/Functions/FunctionsRegexp.cpp

```diff
@@ -23,8 +23,7 @@ namespace DB
 /** Replace all matches of regexp 'needle' to string 'replacement'. 'needle' and 'replacement' are constants.
   * 'replacement' could contain substitutions, for example: '\2-\3-\1'
   */
-template <bool replace_one = false>
-struct ReplaceRegexpImpl
+struct RegexpReplaceImpl
```

Comment:
Looks like `RegexpReplaceImpl` is not used anymore, I think we can just delete it.